familyconnect_registration/settings.py

```diff
@@ -217,10 +217,16 @@
     },
 }
 
+LANGUAGES = ["eng_UG", "cgg_UG"]
```

Comment:
It seems like our `is_valid_lang` has more language options than this. Should we maybe look at adding those options here, and making the `is_valid_lang` function use this? That way we have a single source of truth (because otherwise adding languages here won't make a difference, because `is_valid_lang` will return false and the metric won't fire).